src/diffusers/models/attention_processor.py

```diff
@@ -56,7 +56,8 @@ def __init__(
         bias=False,
         upcast_attention: bool = False,
         upcast_softmax: bool = False,
-        cross_attention_norm: bool = False,
+        cross_attention_norm: Union[bool, str] = False,
+        cross_attention_norm_num_groups: int = 32,
```

Comment:
iirc, group norms in attention blocks for ldm codebases/variants almost always use 32 groups. We probably won't have to actually set this value often